content/hi/docs/reference/glossary/replica-set.md

```diff
@@ -0,0 +1,21 @@
+---
+title: रेप्लिकासेट (ReplicaSet)
+id: replica-set
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/replicaset/
+short_description: >
+  रेप्लिकासेट सुनिश्चित करता है कि एक समय में एक निर्दिष्ट संख्या में पॉड की प्रतिकृतियां चल रही हैं।
```

Comment:
I think that this way the sentence is little easier to understand. Thoughts, WDYT @Pradumnasaraf ? (I do want other contributors/reviewers to have a look at this suggestion too)
```suggestion
  रेप्लिकासेट सुनिश्चित करता है कि पॉड प्रतिकृतियों की एक निर्दिष्ट संख्या एक समय में चल रही हैं।
```